core/shared/src/main/scala/zio/Chunk.scala

```diff
@@ -1762,9 +1784,180 @@ object Chunk extends ChunkFactory with ChunkPlatformSpecific {
 
     override private[zio] def reverseArrayIterator[A1 >: Boolean]: Iterator[Array[A1]] =
       arrayIterator
+  }
+
+  private[zio] final case class BitChunk(bytes: Chunk[Byte], minBitIndex: Int, maxBitIndex: Int)
```

Comment:
Maybe this should be `BitChunkByte` and then `BitChunkBase` can just become `BitChunk`?

Reply:
I tried that and got some "Binary incompatibility" errors. But maybe that's ok, if it only goes in zio 2.x?

Reply:
Yes that is one of the advantages of targeting the `series/2.x` branch.